### ISSUE TITLE
Create env typo?

### ISSUE THREAD

**Issue by jacobjahnkeES** [NREL/Marmot#80]
I am a new user to Marmot and am going through the install here: https://nrel.github.io/Marmot/get-started/install.html
I see the line to create the conda env `conda env create -f marmot-env10.yml` but that file is not in the repo as far as I can tell. There is the file [marmot-linux.yml](https://github.com/NREL/Marmot/blob/main/marmot-linux.yml) and [requirements.txt](https://github.com/NREL/Marmot/blob/main/requirements.txt) and I see the readme says to use the requirements if you are not a linux user. I am raising the issue to see if the documentation needs to be updated.

**Comment by llavin13** [comment]
So, I think we are aware of this issue but @MartySchwarz and I never got around to fixing it. I think my question is ultimately for @micahpw-nrel... are we planning on keeping this repo up to date? If so one of us will make time to fix this. If the plan is to transition to a different open-source plotting tool, I'll let @micahpw-nrel weigh in.

**Comment by jacobjahnkeES** [comment]
Thanks @llavin13, I see now this hasn't been updated in a while. I arrived here by looking at [GridSight](https://github.com/NREL/GridSight/blob/main/docs/tutorials/self-service.MD) which uses marmot.scenariohandlers. Again, I am just getting started here and this seems like an interesting project I'd to explore more!

**Comment by micahpw-nrel** [comment]
@jacobjahnkeES The "scenariohandlers" module is part of another plotting tool that I've been developing but it is currently not open source yet. I've put in a request to open source the GAT (Grid Analysis Toolkit) project here on Github. I will notify you when that is available.

@llavin13 I actually don't know very much about this codebase. @MartySchwarz  and I discussed integrating my plotting tool into Marmot but decided it was best to keep them as separate repositories. As such, I have no intention of maintaining the Marmot repository.

**Comment by jacobjahnkeES** [comment]
@micahpw-nrel thank you!

**Comment by MartySchwarz** [comment]
@jacobjahnkeES thanks for reaching out. `marmot-env.yml` is a conda environment file I made years ago to get Marmot working on my Mac - but it must have since been deprecated. I'll change the language to point to the `marmot-linux.yml` and `environment.txt` file instead. Did you have success just using the `requirements.txt` file to set up your environment? That is definitely the more robust method, depending on your OS. What OS are you using?

Unfortunately I do not have budget to continue maintaining this repo. However, its original author, @daniellevie , may be interested in doing so.

**Comment by MartySchwarz** [comment]
In short, please just follow the installation instructions in the ReadME, and ignore the instructions in [](https://nrel.github.io/Marmot/get-started/install.html). Sorry for the typo! I'll close this issue for now, @jacobjahnkeES please let me know if you're still having issues.

Thanks for using the tool! Would you mind sharing a bit about what you have been using it for, what organization you're with and what kind of models you are running?

**Comment by jacobjahnkeES** [comment]
Thank you @MartySchwarz. I was able to set up the environment, I had to change/remove some versions in the requirements.txt file. I got some things more or less working by trying out some examples. I saw it by looking into GridSight and noticed mention to it there. We do some modelling with Plexos and I am looking into ReEDS so thought it might be a useful tool if I was able to get it working. Thanks again!

**Comment by MartySchwarz** [comment]
@jacobjahnkeES  sounds good, glad you got it working! Package dependencies and their innumerable versions is honestly the bane of my existence. Unfortunately without consistent dedicated maintenance support for Marmot, it seems like these sorts of dependency errors may be inevitable. Which is disappointing because I did spent a good amount of effort back in the day developing Marmot and it has supported a lot of quality research publications over the years.

However, with @micahpw-nrel  expertise GridSight is also a fantastic option. At the outset it was envisioned as a more streamlined tool to see results quickly, whereas Marmot had nice pre-programmed plots that could slot directly into a publication. I'm not sure how things stand now, however, as I have transitioned to a new role in hydrogen research at NREL.
